### PR TITLE
make tv/hours have a background image

### PR DESCRIPTION
**Pull request by abizer**
this looks nice, IMHO

demo at nyx.ocf.berkeley.edu:8934/tv/hours

### CODE REVIEW

File: ocfweb/static/scss/pages/_tv.scss [hunk 1]
```diff
@@ -4,7 +4,14 @@ $tv-hours-small: 5;
 
 .page-tv {
```
Comment (kpengboy) [inline ocfweb/static/scss/pages/_tv.scss]: Does it work to add these attributes to the `body` rule instead?

Comment (abizer) [reply ocfweb/static/scss/pages/_tv.scss]: You mean for the TV's base.html `body`? It does IIRC but .page-tv is on a container div immediately inside body which IMHO is slightly cleaner.

**Comment by matthew-mcallister** [comment]
Isn't this originally an SVG? Can we use that instead of the render?

**Comment by abizer** [comment]
I pulled this from RT where it's a PNG. Raymond hasn't uploaded an SVG yet.

**Comment by raymondnumbergenerator** [comment]
i uploaded the .ai file to rt but i think the image won't save properly as an svg

**Pull request merged**